cis.py

```diff
@@ -90,7 +92,7 @@ def main(argv=None):
                         'See Sample Config File for More information')
 
     proj_work_dir = op.join(args.work_dir,
-                            config_options['project'])
+                            '{0}'.format(config_options['project']))
```

Comment:
No need to do this. `config_options['project']` should already be a string, so there's no need to format a string with the value from that field.

Reply:
fixed